Ports/ncdu/package.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=ncdu
-useconfigure="true"
-version="1.16"
-files="https://dev.yorhel.nl/download/ncdu-${version}.tar.gz ncdu-${version}.tar.gz 2b915752a183fae014b5e5b1f0a135b4b408de7488c716e325217c2513980fd4"
+useconfigure=true
+version=1.17
+files="https://dev.yorhel.nl/download/ncdu-${version}.tar.gz ncdu-${version}.tar.gz 810745a8ed1ab3788c87d3aea4cc1a14edf6ee226f764bcc383e024ba56adbf1"
 auth_type=sha256
 depends=("ncurses")
 
-export CFLAGS="-I${SERENITY_INSTALL_ROOT}/usr/local/include/ncurses"
+export CFLAGS="-I$SERENITY_INSTALL_ROOT/usr/local/include/ncurses"
```

Comment:
Is this even required at all anymore? `ncurses` has been modified recently to install its headers practically everywhere.

Reply:
Yes, it seems this is no longer required.